### DISCUSSION TITLE
Using the rich information that is discovered by AIOPioneer?

### DISCUSSION THREAD

**Discussion by D-Jeffrey** [crowbarz/aiopioneer#19] [General]
When aiopioneer goes thru it's discovery stage, it find about the full configuration of your stereo.  Does anyone have any ideas how that can be used - troubleshooting, improving setup configuration, etc?

I never really understood my Zone 2 configuration capabilities until I saw it show up in the logs.  I have had it so many years, I never realized, I could have separate configuration/sound for another set of speakers.

What have you done with the information you found?

**Comment by pantherale0** [comment]
I think the most I've used of the new features since 0.2.0 were the listening modes and as a one off changing the PQLS settings.

I have far too much spare time on my hands, and got rather frustrated with the Pioneer app not working at all on my phone (Google Pixel, Android 13) so tried to implement as many features as possible so I could use them in Home Assistant.

I do however use the HDZone output to output a signal from HDMI inputs which can then be delivered to any TV screen in the house (I have the Wii attached to the AVR, which then can be used on different TVs thanks to wireless controllers, and those cheap USB sensor bars you can get on Amazon)

I was wondering if I could use Zone 2/3 to output some RCA audio from a Chromecast Audio to some ceiling speakers in other rooms of the house. The project I'm working on now though is mostly spent improving this more than anything. I develop in dotnet for work, so writing some Python is kinda relaxing and soothing haha.
